inngest/deploy.go

```
@@ -91,7 +91,15 @@ func prepareAndPushImage(ctx context.Context, a deployImageOptions) (err error)
 	}
 
 	if a.image == "" {
-		return fmt.Errorf("no image specified")
+		return errors.New("no image specified")
+	}
+
+	resp, _, err := dkr.ImageInspectWithRaw(ctx, a.image)
+	if err != nil {
+		return errors.New("could not locate image")
```

This should return the actual error